### DISCUSSION TITLE
Issues with `t_string` macro

### DISCUSSION THREAD

**Discussion by fabianboesiger** [Baptistemontan/leptos_i18n#151] [Q&A]
Hello there!

I'm trying to use `t_string!(i18n, key)`but I'm getting the following compile time error:

```
no method named `build_string` found for struct `DisplayBuilder` in the current scope
method `build_string` is available on `&str`
```

I can't seem to figure out what I'm missing, maybe a feature? The version I'm using is:

```
leptos_i18n = { version = "0.4.1", features = ["interpolate_display", "track_locale_files"] }
```

Thanks for any help!

**Comment by Baptistemontan** [comment]
This is an issue, I'm able to reproduce it too, I opened #152

**Comment by Baptistemontan** [comment]
Just released `v0.4.2` with the fix
